file-system.ts

```diff
@@ -56,14 +56,19 @@ export class FileSystem implements IFileSystem {
 		return result;
 	}
 
-	public unzip(zipFile: string, destinationDir: string, options?: { overwriteExisitingFiles: boolean}, fileFilters?: string[]): IFuture<void> {
+	public unzip(zipFile: string, destinationDir: string, options?: { overwriteExisitingFiles?: boolean; caseSensitive?: boolean}, fileFilters?: string[]): IFuture<void> {
 		return (() => {
 			var shouldOverwriteFiles = options ? options.overwriteExisitingFiles : true;
+			var isCaseSensitive = options ? options.caseSensitive : true;
```

Comment:
It should be `var isCaseSensitive = options && options.caseSensitive ? options.caseSensitive : true;`